tests/test_utils/test_validators.py

```diff
@@ -137,6 +139,21 @@ class DummySchema2(BaseSchema):
         DummySchema2()
 
 
+@pytest.mark.parametrize(
+    "folder_key,raises_error", [("data", False), ("input_data", False), (None, False), ("i_do_not_exist", True)]
+)
+def test_ensure_storage_key_presence(config, folder_key, raises_error):
```

Comment:
what is `config`?

Reply:
It's a fixture for a global config, containing all managers information. here I use it to bring in the storage structure information, which better simulates what happens in actual pipelines

Reply:
yes, exactly. would you prefer the structure is more explicitly set? it could load the config to get all the necessary pipeline parameters, but then explicitly reset the storage structure

Reply:
in the end I just explicitly updated the `config` to 

```
config["storage"]["structure"] = {"i_exist": "foobar"}
```

because adding a `dict` param to the dummy schema doesn't work, since the validator works with a storagemanager schema object. I could convert it to a dict, but that already seemed like cooking.

Reply:
ah right, should be fine now